astropy/io/ascii/fixedwidth.py

```diff
@@ -312,12 +312,13 @@ class FixedWidthNoHeaderData(FixedWidthData):
 
 
 class FixedWidthNoHeader(FixedWidth):
-    """
-    Read or write a fixed width table which has no header line.  Column
-    names are either input (``names`` keyword) or auto-generated.  Column
-    positions are determined either by input (``col_starts`` and ``col_stops``
-    keywords) or by splitting the first data line.  In the latter case a
-    ``delimiter`` is required to split the data line.
+    """Fixed width table which has no header line.
```

Comment:
Yes, I have thought in past about some way to do this, but indeed that's a whole different beast and not entirely trivial.

Reply:
Could you open an issue about it, so we know about it? I assume this may be a problem for other packages, so will try to keep this in mind in case I run into a solution.

Reply:
#8578